lib/approvals/configuration.rb

```diff
@@ -15,8 +15,7 @@ def configuration
   class Configuration
     include Singleton
 
-    attr_writer :approvals_path
-    attr_writer :excluded_json_keys
+    attr_writer :approvals_path, :excluded_json_keys, :default_format
```

Comment:
I like this style less, because of risk of incidental change:
http://www.rubytapas.com/episodes/24-Incidental-Change?filter=free